coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,58 @@
+from coala_quickstart.Strings import coala_bear_logo, welcome_messages
+
+
+def text_wrap(*args, delimiter=' ', end='\n', limit=80):
+    """
+    Breaks it word wise when going further than ``limit`` chars.
+
+    :param args:
+        The stuff you want to have printed.
+    :param delimiter:
+        Will be placed between all the args.
+    :param limit:
+        Char limit.
+    :param end:
+        An ending, will be put as last character and doesn't count into the
+        char limit.
+    """
+    output = delimiter.join(args)
+    lines = output.splitlines(keepends=True)
+    results = [""]
+    for line in lines:
+        curr_print = line
+        while len(curr_print.rstrip('\n')) > limit:
+            splitpos = curr_print[:limit].rfind(' ')
+            if splitpos < 0:
+                # Word too long, search for a space left from limit at least
+                splitpos = curr_print.find(' ')
+                if splitpos < 0:
+                    break  # Break out and print the long thing in 28
+
+            results.append(curr_print[:splitpos])
+            curr_print = curr_print[splitpos+1:]
+
+        results.append(curr_print)
+
+    return results
+
+
+def print_welcome_message(printer):
+    """
+    Prints the coala bear logo alongside a welcome message.
+
+    :param printer:
+        A ``ConsolePrinter`` object used for console interaction.
+    """
+    max_length = 80 - len(max(coala_bear_logo, key=len))
+
+    text_lines = [""]
+    for welcome_message in welcome_messages:
+        text_lines += text_wrap(welcome_message, limit=max_length)
+
+    for line in range(len(coala_bear_logo) - len(text_lines)):
+        text_lines.append("")
```

Comment:
https://i.imgur.com/if1nR1a.png with a tiny modification to append `""` lines when the text is longer than the logo